shoes-swt/lib/shoes/swt/common/painter.rb

```diff
@@ -98,7 +98,7 @@ def reset_rotate(transform, graphics_context, angle, left, top)
           transform.translate left, top
           transform.rotate angle
           transform.translate(-left, -top)
-          graphics_context.setTransform transform
+          graphics_context.set_transform transform
```

Comment:
Yeah noticed that as well. Probably will turn them all to = form as long as it goes smoothly next time I'm coding.